tensorflow/training/docker/2.10/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,300 @@
+FROM ubuntu:20.04 AS common
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# TensorFlow major.minor version 
+ENV TF_VERSION=2.10
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+
+ENV KMP_AFFINITY=granularity=fine,compact,1,0
+ENV KMP_BLOCKTIME=1
+ENV KMP_SETTINGS=0
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+ARG PYTHON=python3.9
```

Comment:
Is there a reason we are using Python 3.9 specifically? TF 2.10 supports up to Python 3.10 (https://www.tensorflow.org/install/pip#software_requirements)

Reply:
This was a group decision made collectively by the relevant AWS stakeholder teams at the beginning of the TF 2.10 release process.  I believe the main reason was because there was no specific advantage cited for moving away from Python 3.9, and we wanted to minimize changes to dependencies carried over from TF 2.9.

Reply:
@dkey-amazon does Python3.10 support is on the roadmap?

Reply:
> @dkey-amazon does Python3.10 support is on the roadmap?

There are not currently plans to release a Python 3.10 AWS deep learning container for Tensorflow 2.10.  Did you have a particular use case in mind @AlonAvrahami?